src/constants.ts

```diff
@@ -0,0 +1,26 @@
+/* terra:network */
+export const FINDER = "https://finder.terra.money";
+export const EXTENSION = "https://terra.money/extension";
+export const CHROME = "https://google.com/chrome";
+export const DOCS = "https://docs.freewilly.money";
+
+/* terra:wasm */
+export const WASMQUERY = "WasmContractsContractAddressStore";
+
+/* terra:configs */
+export const BLOCK_TIME = 6500; // 6.5s
+
+/* mirror:unit */
```

Comment:
Do we need any of the mirror specific settings?  Any we can remove, or should we just keep them all?